browserassets/html/traitorTips/vampireTips.html

```diff
@@ -15,7 +15,7 @@ <h1 class="center">You are a vampire!</h1>
  		<span class="small"><br><br>
  			<span style="float:left;width:50%;">
 				40 - <em>Bat Form</em>, consume stamina to morph into a bat.<br>
-				40 - <em>Enthrall / Thrall Speak</em>, revive a dead human as an enthralled servant.<br>
+				40 - <em>Enthrall / Thrall Speak / Alert Thralls</em>, revive a dead human as an enthralled servant.<br>
```

Comment:
Wouldn't it be better if alert thralls had it's own mention, rather than being lumped in without mention?